server/routes/user/user.js

```diff
@@ -26,15 +26,18 @@ router.put('/user/avatar', authentication, async (req, res) => {
     const userId = req.user.id;
     const { avatar } = req.body;
 
+    console.log('this is from be', req.files.avatar);
+
     const user = await User.findById(userId);
     console.log(user.name);
     if (!user) return res.status(400).json({ msg: 'User not found' });
 
     console.log(req.body.avatar);
-    const uploadAvatar = await cloudinary.uploader.upload(req.avatar.tempFilePath, {
+    const uploadAvatar = await cloudinary.uploader.upload(req.files.avatar.tempFilePath, {
       upload_preset: 'team_pumpkin',
     });
-    console.log(uploadAvatar.url);
+    // image processed by the cloudinary server and this is where you need to send the url to front end and save to db as well/
```

Comment:
follow the comment @TheAllen also you need to style the fe when you get the url back